steamos-waydroid-installer.sh

```diff
@@ -291,7 +294,22 @@ else
 		elif [ "$Choice" == "TV" ]
 		then
 			echo Android TV chosen!
-			exit
+			echo Initializing Waydroid
+			echo -e "$current_password\n" | sudo -S waydroid init
```

Comment:
```suggestion
```

removing the duplicative initial init per [above comments](https://github.com/ryanrudolfoba/SteamOS-Waydroid-Installer/pull/220/files#r1911014154)